openassessment/templates/openassessmentblock/staff_area/oa_staff_grade_learners_assessment.html

```diff
@@ -0,0 +1,58 @@
+{% load i18n %}
+{% spaceless %}
+
+    {% block body %}
+    <div class="staff__grade__form ui-toggle-visibility__content" data-submission-uuid="{{ submission.uuid }}">
+        <div class="wrapper--staff-assessment">
+            <div>
+                <p>{% trans "Give this learner a grade using the problem's rubric." %}</p>
+            </div>
+
+            <div>
+                <article class="staff-assessment">
+                    <div class="staff-assessment__display">
+                        <header class="staff-assessment__display__header">
+                            <h4 class="staff-assessment__display__title">
+                                {% blocktrans %}
+                                    Response for: {{ student_username }}
+                                {% endblocktrans %}
+                            </h4>
+                        </header>
+
+                        {% include "openassessmentblock/oa_submission_answer.html" with answer=submission.answer answer_text_label="The learner's response to the question above:" %}
+
+                        {% include "openassessmentblock/oa_uploaded_file.html" with file_upload_type=file_upload_type file_url=staff_file_url header="Associated File" class_prefix="staff-assessment" show_warning="true" %}
+                    </div>
+
+                    <form class="staff-assessment__assessment" method="post">
+                        {% include "openassessmentblock/oa_rubric.html" with rubric_type="staff" rubric_feedback_prompt="(Optional) What aspects of this response stood out to you? What did it do well? How could it improve?" rubric_feedback_default_text="I noticed that this response..." %}
```

Comment:
@cahrens The wording that we came up with for self and peer grading should be used here too, right? (TNL-3882) 
In the last sentence of the instructions, change "How could it improve?" to "How could it be improved?"
Change the default feedback example from "I noticed that this response..." to "I think that this response..."